fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/materialization/rule/BaseMaterializedViewRewriteRule.java

```diff
@@ -96,7 +96,7 @@ public List<OptExpression> transform(OptExpression queryExpression, OptimizerCon
         try {
             return doTransform(queryExpression, context);
         } catch (Exception e) {
-            String errMsg = ExceptionUtils.getMessage(e);
+            String errMsg = ExceptionUtils.getStackTrace(e);
```

Comment:
That would generate a huge error message, which may make it un-readable ?

Reply:
This should not happen and  trace log could help for trace the root cause?